server/db/model/userloan.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (sequelize, type) => sequelize.define('userLoan', {
```

Comment:
I didnt think we were saving anything to database. Are you going anything with this code here client side?